apps/client/src/pages/Auth/Login/index.tsx

```diff
@@ -36,6 +37,23 @@ const LoginPage = () => {
     if (data) handleAuthSession(data.data)
```

Comment:
handleAuthSession(data.data) is throwing an error on data.data from my end. I don't want to touch it. I just wanted to let you know.